src/app/content/components/Page/PageNotFound.tsx

```diff
@@ -33,6 +34,17 @@ const PageNotFoundText = styled.div`
   }
 `;
 
+// tslint:disable-next-line: variable-name
+export const StyledOpenSidebarControl = styled(OpenSidebarControl)`
+  min-height: unset;
+  flex-direction: row;
+  justify-content: start;
+
+  ${ToCButtonText} {
+    ${textRegularSize};
+  }
+`;
```

Comment:
I think it would make more sense to store this in SidebarControl in case we want to reuse it elsewhere

Reply:
@bethshook 
You mean only the different ToCButtonText styling, am i right? Or the whole component that makes the toc button icon and the text to be next to each other?
![obraz](https://user-images.githubusercontent.com/36712200/144881893-5e3016d4-7901-4fe5-9277-dc1767bfc7c5.png)

Reply:
I was thinking the whole component